src/wp-content/themes/twentytwentytwo/functions.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Twenty Twenty-Two functions and definitions
+ *
+ * @link https://developer.wordpress.org/themes/basics/theme-functions/
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Two
+ * @since 1.0.0
+ */
+
+
+if ( ! function_exists( 'twentytwentytwo_support' ) ) :
+
+	/**
+	 * Add support for core block visual styles.
+	 * Styles load in both the editor and the front end.
+	 *
+	 * @link https://developer.wordpress.org/block-editor/how-to-guides/themes/theme-support/#default-block-styles
+	 *
+	 * @return void
+	 */
+	function twentytwentytwo_support() {
+
+		add_theme_support( 'wp-block-styles' );
+
+	}
+	add_action( 'after_setup_theme', 'twentytwentytwo_support' );
+
+endif;
+
+
+if ( ! function_exists( 'twentytwentytwo_styles' ) ) :
+
+	/**
+	 * Enqueue styles.
+	 *
+	 * @return void
+	 */
+	function twentytwentytwo_styles() {
+
+		// Register theme stylesheet.
+		wp_register_style(
+			'twentytwentytwo-style',
```

Comment:
`twentytwentytwo` is more like the "namespace", and it feels conventional and more specific to append the `-style` here, since traditionally you might have multiple stylesheets (e.g. an additional print).

Reply:
Personally, I feel the `-style` is redundant, and adding `-print`, etc. to the handle is better. But it makes no difference in the end, and not a hill I'm willing to die on. Looks like TT1 had `-style` as well, so fine to leave this as is.